Add "az acr list" link.

anikundesu opened the pull request:
Add "az acr list" link to the document written in issue ( https://github.com/MicrosoftDocs/mvp-hackadoc-2018/issues/3 )

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

Byronr commented:
@alexandair Thanks for the updates. @ Tysonn Please merge.

alexandair commented:
@byronr @tysonn After 24 days this is still not merged. :(

anikundesu committed:
Add "az acr list" link.

Add "az acr list" link to the document written in issue ( https://github.com/MicrosoftDocs/mvp-hackadoc-2018/issues/3 )
